### PR TITLE
Restructure and update contributing guides

### PR DESCRIPTION
**Pull request by charislam**
# Description

Contributing and style information is spread between multiple places: two contributing pages on the docs site, `README.md`, and `CONTRIBUTING.md`. This PR reorganizes the information to make it more discoverable and less overlapping, so others find it easier to contribute to our docs.

### `CONTRIBUTING.md`
- Restructure to include Github workflow information, repository structure, and markup conventions. Making this information available in one place makes it less overwhelming to get started
- Rewrite GitHub workflow instructions to generally follow docs conventions on procedures
- Remove `<highlight>`s as they don't display in GitHub, and the file isn't viewable from the docs site
- Remove deprecated syntax from markup conventions
- Clarify instructions for internal links with examples
- Add more information on `<highlight>`, `<procedure>`, and `<tag>` components

### `README.md`
- Remove repository structure and markup convention information (moved to `CONTRIBUTING.md`)
- Remove outdated information about branches and version numbers

### Docs site
- Remove repeated sections from contributing pages on docs site
- Restructure style guide
- Move contributing into top-level section on docs site
- Additions to style guide:
  - Please
  - We
  - Numbers: write 0 to 9 as numerals too for readability (small exception from Google's style guide, see [Slack conversation](https://iobeam.slack.com/archives/C0J94TE4F/p1646945388271069))

### PR request
- Remove version selector as it doesn't do anything in current build process

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

**Comment by charislam** [comment]
@Loquacity @jacobprall I'd like to move the Contributing pages into their own top-level section unless you feel there's a good reason not to. I think they will be more discoverable at the top level.

I changed the `page-index.js` files to move them, and I checked that the URLs work as expected. But they don't appear in the nav. How do I add a new top-level section to nav?

File: contribute-to-timescaledb/contribute-to-docs.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
+# Contribute to Timescale documentation
+Join the community in making Timescale documentation even better. You can
```
Comment (Loquacity) [inline contribute-to-timescaledb/contribute-to-docs.md:2]:
```suggestion
Join the community to help make Timescale documentation even better. You can
```

File: contribute-to-timescaledb/contribute-to-docs.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+contribute to the [documentation repository on GitHub][github-docs].
+
+Issues and pull requests of any size, from typo fixes to new content sections,
+are highly welcome.
```
Comment (Loquacity) [inline contribute-to-timescaledb/contribute-to-docs.md:6]:
```suggestion
are always welcome.
```

File: contribute-to-timescaledb/contribute-to-docs.md [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+ready. 
+
+To learn more:
+*   Keep reading this style guide, which covers the documentation's use of the
+    English language
+*   See the [contribution guide][contributors] to learn about the GitHub
+    workflows, repository structure, and markup conventions
+
+## Style guide sections
+
```
Comment (Loquacity) [inline contribute-to-timescaledb/contribute-to-docs.md:20]:
```suggestion
```

File: contribute-to-timescaledb/contribute-to-docs.md [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+    *   Style references in order of importance
+*   [Language](#language)
+    *   Plain English
+    *   Tense and perspective
+    *   Voice
+    *   Grammar
+    *   User interface actions
+    *   Prose linting with Vale
+*   [Structure](#structure)
+    *   Information when it's needed, where it's needed
```
Comment (Loquacity) [inline contribute-to-timescaledb/contribute-to-docs.md:32]:
```suggestion
    *   Information where it's needed, when it's needed
```

File: contribute-to-timescaledb/contribute-to-docs.md [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+|✅ Plain English|To install X, do these steps ...|
+
+### Tense and perspective
+Use simple present tense and the second person singular ("you"). Avoid the first
+person plural ("we," "us"), because it isn't clear who is included in that
+group.
+
+### Voice
+We prefer active voice, but don't be afraid to use passive when it serves
+better. For example, the most important words come first in an English sentence.
```
Comment (Loquacity) [inline contribute-to-timescaledb/contribute-to-docs.md:78]:
```suggestion
the purpose better. For example, the most important words come first in an English sentence.
```

File: contribute-to-timescaledb/contribute-to-docs.md [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+If the object of an action is the most important part of a sentence, put it
+first, even if it means using passive.
```
Comment (Loquacity) [inline contribute-to-timescaledb/contribute-to-docs.md:80]:
```suggestion
first, even if it means using passive voice.
```

File: contribute-to-timescaledb/contribute-to-docs.md [hunk 7]
```diff
@@ -0,0 +1,266 @@
...
+
```
Comment (Loquacity) [inline contribute-to-timescaledb/contribute-to-docs.md:81]: I have a way of explaining this where I point out that, in technical documentation, we almost always know who the actor is (it's the reader, "you"), so in most cases the action is actually the most important part of the sentence ("Install the thing", "Configure the thing", etc). Feel free to construct that into a more useful piece of advice here, if you think it's useful 😉

File: contribute-to-timescaledb/contribute-to-docs.md [hunk 8]
```diff
@@ -0,0 +1,266 @@
...
+### Tense and perspective
+Use simple present tense and the second person singular ("you"). Avoid the first
+person plural ("we," "us"), because it isn't clear who is included in that
+group.
+
+### Voice
+We prefer active voice, but don't be afraid to use passive when it serves
+better. For example, the most important words come first in an English sentence.
+If the object of an action is the most important part of a sentence, put it
```
Comment (Loquacity) [inline contribute-to-timescaledb/contribute-to-docs.md:79]:
```suggestion
If the object or the action is the most important part of a sentence, put it
```

File: contribute-to-timescaledb/contribute-to-docs.md [hunk 9]
```diff
@@ -0,0 +1,266 @@
...
+*   Errors. These block merging of your pull request into the official published
+    docs. They must be fixed before your contribution can be accepted.
+*   Warnings. Less serious than errors. Adopt these recommendations unless you
+    have a strong reason not to.
+*   Suggestions. The least serious level. Consider and adopt these
+    recommendations, unless your content is a special case or exception.
+
+## Structure
+
+### Information when it's needed, where it's needed
```
Comment (Loquacity) [inline contribute-to-timescaledb/contribute-to-docs.md:119]:
```suggestion
### Information where it's needed, when it's needed
```

File: contribute-to-timescaledb/contribute-to-docs.md [hunk 10]
```diff
@@ -0,0 +1,266 @@
...
+Readers don't read documentation from cover to cover, like a book. They skim and
+skip. Don't assume that they know something because it's explained earlier in
+the document. Use cross-references to guide them to further information when
+they need it.  
+
```
Comment (Loquacity) [inline contribute-to-timescaledb/contribute-to-docs.md:124]: Perhaps add a reference to Every Page is Page One here? Or is that going too much into the content strategy weeds?

File: contribute-to-timescaledb/contribute-to-docs.md [hunk 11]
```diff
@@ -0,0 +1,266 @@
...
+## V [](#V)
+
+*Via*
+: Avoid if possible. There is usually a more accurate English word, like
+"through," "with," or "using."
+
+## W [](#W)
+
+*We*
+: Do not use. Use second person ("you") instead.
```
Comment (Loquacity) [inline contribute-to-timescaledb/contribute-to-docs.md:244]: I think we can safely say to use this judiciously, and only when we're referring to Timescale (the company). As in, "We strongly recommend you upgraded to the newest version of TimescaleDB", etc. I think it's better to try adn avoid it, generally, but there are definitely places where I would prefer to use "we" over "Timescale" to suit our company voice better.

File: contribute-to-timescaledb/index.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
+# Contribute to TimescaleDB
+Join the community in making TimescaleDB even better. You can contribute to the
```
Comment (Loquacity) [inline contribute-to-timescaledb/index.md:2]:
```suggestion
Join the community ito make TimescaleDB even better. You can contribute to the
```

File: contribute-to-timescaledb/index.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+[TimescaleDB source code][github-timescaledb] and [documentation][github-docs],
+both of which available on GitHub. Issues and pull requests are highly welcome.
```
Comment (Loquacity) [inline contribute-to-timescaledb/index.md:4]:
```suggestion
both of which are available on GitHub. Issues and pull requests are always welcome.
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -1,111 +1,179 @@
 # Introduction
-There are multiple ways to help make TimescaleDB better. All of our
-documentation and source for the PostgreSQL extension are available to use and
-review with GitHub.
+Join the community in making TimescaleDB documentation better. Issues and pull
+requests are highly welcome.
```
Comment (charislam) [inline CONTRIBUTING.md:3]:
```suggestion
requests are always welcome. 
```

File: timescaledb/page-index/page-index.js [hunk 15]
```diff
@@ -20,6 +20,20 @@ module.exports = [
       ...howtoPageIndex,
       ...tutorialPageIndex,
       ...quickStartPageIndex,
+      {
+        title: 'Contribute to TimescaleDB',
+        href: 'contribute-to-timescaledb',
+        tags: ["timescaledb", "overview", "get started", "learn"],
+        keywords: ["code", "GitHub", "git", "error"],
+        excerpt:"Learn more about how to contribute to TimescaleDB",
```
Comment (Loquacity) [inline timescaledb/page-index/page-index.js:28]:
```suggestion
        excerpt:"Learn how to contribute to TimescaleDB",
```

File: timescaledb/page-index/page-index.js [hunk 16]
```diff
@@ -20,6 +20,20 @@ module.exports = [
       ...howtoPageIndex,
       ...tutorialPageIndex,
       ...quickStartPageIndex,
+      {
+        title: 'Contribute to TimescaleDB',
+        href: 'contribute-to-timescaledb',
+        tags: ["timescaledb", "overview", "get started", "learn"],
```
Comment (Loquacity) [inline timescaledb/page-index/page-index.js:26]:
```suggestion
        tags: ["contribute", "get started", "timescaledb"],
```

File: timescaledb/page-index/page-index.js [hunk 17]
```diff
@@ -20,6 +20,20 @@ module.exports = [
...
+        keywords: ["code", "GitHub", "git", "error"],
+        excerpt:"Learn more about how to contribute to TimescaleDB",
+      },
+      {
+        title: 'Contribute to documentation',
+        href: 'contribute-to-docs',
+        tags:["timescaledb", "get started", "learn", "contribute"],
```
Comment (Loquacity) [inline timescaledb/page-index/page-index.js:33]:
```suggestion
        tags:["contribute", "documentation", "get started"],
```

File: timescaledb/page-index/page-index.js [hunk 18]
```diff
@@ -20,6 +20,20 @@ module.exports = [
...
+        keywords:["code", "docs", "markdown", "style guide"],
+        excerpt:"Learn more about how to contribute to TimescaleDB documentation",
```
Comment (Loquacity) [inline timescaledb/page-index/page-index.js:35]:
```suggestion
        excerpt:"Learn how to contribute to Timescale documentation",
```

**Review by Loquacity — APPROVED** [review]

**Comment by Loquacity** [comment]
This doesn't seem to be working in preview.

File: CONTRIBUTING.md [hunk 19]
```diff
@@ -293,7 +293,7 @@ Each page listed in `page-index.js` can have the following properties:
 |`children`|Optional|An array containing the child pages for the page. Child-page properties are defined in the same way as parent-page properties. Child pages can be nested inside other child pages to form multiple levels. The filenames provided in `href` should be located in a sub-folder with the same name as the parent page.|
 |`keywords`|Optional|An array of keywords to be displayed at the bottom of the page.|
 |`excerpt`|Optional|A short description of the page. Excerpts are displayed within Related Content cards at the bottom of each page.|
-|`relatedPages`|Optional|An array of `href`s to be displayed as Related Content at the bottom of the page.|
+|`relatedPages`|Optional|An array of pathnames to be displayed as Related Content at the bottom of the page. For example, `["/promscale/:currentVersion:/installation/"]`|
```
Comment (Loquacity) [inline CONTRIBUTING.md:296]:
```suggestion
|`relatedPages`|Optional|An array of path names to be displayed as Related Content at the bottom of the page. For example, `["/promscale/:currentVersion:/installation/"]`|
```

**Comment by charislam** [comment]
We will not be moving contributing to top level in near future. Going to take this PR apart into several parts to make some other improvements to our contributing docs.

**Pull request closed without merging**